docs/source/quicktour.rst

```diff
@@ -125,16 +132,19 @@ While you review the publications that the software presents you with,
 the software continuously improves its understanding of your decisions,
 constantly updating the underlying model.
 
-.. figure:: ../images/5_reviewing.png
+.. figure:: ../images/5.1_start_reviewing.png
    :alt:
 
-To view your progress, click the Statistics symbol in the upper right corner.
-The Statistics panel will open, showing information on the current
-project and labelling progress.
 
-.. figure:: ../images/statistics_1.png
+The Statistics panel shows information on the current project and labelling progress. 
+The panel can be opened and closed during labeling as you like. 
+To hide your progress, click the Statistics symbol in the upper right corner.
```

Comment:
This is the closing arrow, isn't it? Statistics symbol is for panel expansion.

Reply:
Whoops I mixed up some sentences there. Thanks